appveyor.yml

```diff
@@ -31,7 +36,9 @@ install:
 
   # This upates conda and installs the necessary packages.
   - "%PYTHON%\\Scripts\\conda.exe update --yes conda"
-  - "%PYTHON%\\Scripts\\conda.exe install --yes numpy scipy sympy cython pywin32 nose coverage"
+  - "%PYTHON%\\Scripts\\conda.exe config --prepend channels conda-forge"
+  - '%PYTHON%\\Scripts\\conda.exe install --yes numpy scipy sympy cython pywin32 nose coverage'
+  - "%PYTHON%\\Scripts\\conda.exe update -y --all"
```

Comment:
I got all kinds of numpy errors and running this update all command is what solved it.